maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -194,27 +195,8 @@ def post_step(self, tick: int):
 
         return tick + 1 == self._max_tick
 
-    def rewards(self, actions: list):
```

Comment:
Double-check with @chaosddp

Reply:
Can be deleted, as we have no reward for now.